core/frontend/src/components/vehiclesetup/params/roverParams.ts

```diff
@@ -0,0 +1,915 @@
+import { SemVer } from 'semver'
+import { Dictionary } from 'vue-router/types/router'
+
+// This is from the QGC file as provided by Eldin
+// # Vehicle-Id Component-Id Name Value Type
+const bb120params = {
```

Comment:
are all this parameters necessary ? Wasn't the idea to not include everything ?

Reply:
> The blueboat params are currently a full dump, which I expect to cleanup as soon as I'm cleared to do so.

I'm waiting for confirmation that I can clean these up from the bb120 team